### ISSUE TITLE
Can you provide a color palette for semantic segmentation?

### ISSUE THREAD

**Issue by Jacky-Android** [OucCVLab/OUC-UAV-SEG#1]
I need the RGB value for each category

**Comment by XDWang97** [comment]
Thank you for your attention! You are welcome to create your own color palette, though we have provided one below for your reference.

    
# Define colors for different classes (BGR format)
    colors = {
        0: [0, 0, 255],     # Class 0 (red)
        1: [0, 255, 0],     # Class 1 (green)
        2: [255, 0, 0],     # Class 2 (blue)
        3: [0, 255, 255],   # Class 3 (yellow)
        4: [255, 255, 0],   # Class 4 (cyan)
        5: [255, 0, 255],   # Class 5 (magenta)
        6: [128, 0, 0],     # Class 6 (maroon)
        7: [0, 128, 0],     # Class 7 (dark green)
        8: [0, 0, 128],     # Class 8 (dark blue)
        9: [128, 128, 0],   # Class 9 (olive)
        10: [128, 0, 128],  # Class 10 (purple)
        11: [0, 128, 128],  # Class 11 (teal)
        12: [64, 64, 0],    # Class 12 (dark olive)
        13: [64, 0, 64],    # Class 13 (dark purple)
        14: [0, 64, 64],    # Class 14 (dark teal)
        15: [0, 0, 64],    # Class 15 (dark teal)
        # Add more colors for other classes here
    }

    # Define class labels
    class_labels = {
        0: "back",
        1: "seaweed",
        2: "enteromorpha",
        3: "oil",
        4: "sea",
        5: "river",
        6: "land",
        7: "spartina",
        8: "suaeda",
        9: "tamarix",
        10: "reed",
        11: "vegetation",
        12: "building",
        13: "sky",
        14: "road",
        15: "boat",
        # Add more class labels as needed
    }

`

**Comment by Jacky-Android** [comment]
> 感谢您的关注！欢迎您创建自己的调色板，不过我们在下面提供了一个调色板供您参考。
> 
> # 为不同类别定义颜色（BGR 格式）
> ```
> colors = {
>     0: [0, 0, 255],     # Class 0 (red)
>     1: [0, 255, 0],     # Class 1 (green)
>     2: [255, 0, 0],     # Class 2 (blue)
>     3: [0, 255, 255],   # Class 3 (yellow)
>     4: [255, 255, 0],   # Class 4 (cyan)
>     5: [255, 0, 255],   # Class 5 (magenta)
>     6: [128, 0, 0],     # Class 6 (maroon)
>     7: [0, 128, 0],     # Class 7 (dark green)
>     8: [0, 0, 128],     # Class 8 (dark blue)
>     9: [128, 128, 0],   # Class 9 (olive)
>     10: [128, 0, 128],  # Class 10 (purple)
>     11: [0, 128, 128],  # Class 11 (teal)
>     12: [64, 64, 0],    # Class 12 (dark olive)
>     13: [64, 0, 64],    # Class 13 (dark purple)
>     14: [0, 64, 64],    # Class 14 (dark teal)
>     15: [0, 0, 64],    # Class 15 (dark teal)
>     # Add more colors for other classes here
> }
> 
> # Define class labels
> class_labels = {
>     0: "back",
>     1: "seaweed",
>     2: "enteromorpha",
>     3: "oil",
>     4: "sea",
>     5: "river",
>     6: "land",
>     7: "spartina",
>     8: "suaeda",
>     9: "tamarix",
>     10: "reed",
>     11: "vegetation",
>     12: "building",
>     13: "sky",
>     14: "road",
>     15: "boat",
>     # Add more class labels as needed
> }
> ```
> 
> `

Thank you very much!!!

**Comment by Jacky-Android** [comment]
Can we put the test set on the competition website for testing later?

**Comment by XDWang97** [comment]
> Can we put the test set on the competition website for testing later?

Yes, we do plan to organize such events in the future. However, due to the complexity of the annotation process, the current dataset is still limited, making it difficult to support hosting a competition at this stage. We will update a new version of the dataset before the end of the year. Please keep an eye on this repository.
